requirements/edx/base.txt

```diff
@@ -339,7 +340,7 @@ django-sekizai==3.0.1
     # via
     #   -r requirements/edx/base.in
     #   django-wiki
-django-ses==3.0.1
+django-ses==3.1.0
```

Comment:
No breaking change mentioned for the new version. See [CHANGELOG](https://github.com/django-ses/django-ses/blob/master/CHANGES.md#current)